presto-main/src/main/java/com/facebook/presto/sql/planner/plan/JoinNode.java

```diff
@@ -176,7 +187,12 @@ public PlanNode getRight()
     public PlanNode replaceChildren(List<PlanNode> newChildren)
     {
         checkArgument(newChildren.size() == 2, "expected newChildren to contain 2 nodes");
-        return new JoinNode(getId(), type, newChildren.get(0), newChildren.get(1), criteria, filter, leftHashSymbol, rightHashSymbol);
+        return new JoinNode(getId(), type, newChildren.get(0), newChildren.get(1), criteria, outputSymbols, filter, leftHashSymbol, rightHashSymbol);
+    }
+
+    public boolean isNestedLoopJoin()
```

Comment:
This should be `isCrossJoin`. Nested loops is an implementation strategy, which, technically, could be used for any type of join.